docs/guides/rate_limits.md

```diff
@@ -49,20 +49,33 @@ from tenacity import (
 def completion_with_backoff(**kwargs):
     return llmengine.Completion.create(**kwargs)
 
-completion_with_backoff(model="llama-7b", prompt="Why is the sky blue?")
+print(  
```

Comment:
I don't think we actually need to `print` stuff here, since the main thing is to show the retrying behavior, not the actual result. But if we are going to do so, perhaps we should have consistent indention with line 76.